### PR TITLE
job done.

### PR DESCRIPTION
**Pull request by AlexNeiko**

### CODE REVIEW

File: homework0/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+**Hello fron Alex Neiko**
```
Comment (Dmitry404) [inline homework0/readme.md]: Incorrect path to the readme file. Mind your name and surname

File: homework0/Alex.Neiko/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+**Hello fron Alex Neiko**
```
Comment (Dmitry404) [inline homework0/Alex.Neiko/readme.md]: lower case plz

Comment (PeYceBall) [reply homework0/Alex.Neiko/readme.md]: trolled

Comment (Dmitry404) [reply homework0/Alex.Neiko/readme.md]: Всем чмоки в этом чате (ц)

 @AlexNeiko please change to `alex.neiko`

**Comment by Dmitry404** [comment]
Going to close this PR as there are no recent updates

**Pull request closed without merging**